packages/components/psammead-brand/src/index.jsx

```diff
@@ -69,34 +65,26 @@ const StyledSpan = styled.span`
 const BrandSvg = styled.svg`
   display: block;
   height: ${SVG_HEIGHT};
-  width: ${SVG_WIDTH};
   margin-top: ${SVG_TOP_OFFSET};
-`;
-
-const SvgGroup = styled.g`
   fill: #fff;
-
   @media screen and (-ms-high-contrast: active), print {
     fill: windowText;
   }
 `;
 
-const Brand = ({ brandName }) => (
+const Brand = ({ brandName, svg }) => (
   <StyledWrapper>
     <ConstraintWrapper>
       <StyledLink href="https://www.bbc.co.uk/news">
         <StyledSpan>
           <BrandSvg
-            viewBox={VIEWBOX_VALUES}
+            viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height} `}
             xmlns="http://www.w3.org/2000/svg"
             focusable="false"
             aria-hidden="true"
+            width={svg.viewbox.width}
```

Comment:
We should set the height, not the width, as thats what the UX uses

Reply:
It should be passed in as a prop too, so it can be set differently for different services.

Reply:
> We should set the height, not the width, as thats what the UX uses

The height is already set. 
https://github.com/bbc/psammead/blob/225ba060e7068150d6c0032b4cdda336a7f5a77c/packages/components/psammead-brand/src/index.jsx#L65-L73

Reply:
Ok, we should remove that and make it set on a prop as UX requires it can be set dynamically

Reply:
Im happy for us to do the work for a dynamic height in a different PR later, but we should remove `width={svg.viewbox.width}` as its not needed and isnt really truthful as viewport width and width arnt the same

Reply:
Done. `width` removed. The work on dynamic height could be done on a separate PR.